lib/unleash/feature_toggle.rb

```diff
@@ -189,7 +189,7 @@ def ensure_valid_context(context)
     end
 
     def initialize_strategies(params, segment_map)
-      params.fetch('strategies', [])
+      (params.fetch('strategies', []) || [])
         .select{ |s| s.has_key?('name') && Unleash.strategies.includes?(s['name']) }
         .map do |s|
```

Comment:
Personally, this is how i would have solved this same issue, as i find that more legible. But your approach is correct as well (and what was implemented in the other methods in the SDK below).

```suggestion
      params.fetch('strategies', [])
        &.select{ |s| s.has_key?('name') && Unleash.strategies.includes?(s['name']) }
        &.map do |s|
```

So you can implement using the safe navigation operator for bonus points, but otherwise it looks great!

(I will try to get a PR going converting the other methods to also use the safe navigation operator, for increased legibility.

Reply:
works for me!